### DISCUSSION TITLE
Dominos and Masque Button Appearance Sticking

### DISCUSSION THREAD

**Discussion by charleslaine** [SFX-WoW/Masque#169] [Issues] [answered]
I dont know if it's dominos or masque that's doing this.. but with the two addons working together, a lot of times a button will "stick" to the pressed state.  IT is basically showing a solid white icon.  It's like the button is stuck down.  But it's only the skin.  I will post the same msg on both addon's pages.

**Comment by StormFX** [comment, marked as answer]
I use Bartender and haven't experienced a similar issue so I'll assume the problem is/was on Dominos' end.
